store/store.go

```diff
@@ -97,6 +98,7 @@ func GetStoreWithParams(
 	name string,
 	lockTryDuration time.Duration,
 	lockHoldTimeout time.Duration,
+	nameSpace string,
```

Comment:
I feel it's not required to pass in namespace as parameter in all these functions. 
Why not just call `ns := os.Getenv("PX_NAMESPACE")` in newK8sStoreWithParams and pass that to configmap.New().